packages/process/test/exec.test.ts

```diff
@@ -42,6 +42,13 @@ describe('exec', () => {
       expect(error.name).toEqual('ExecError');
     });
 
+    it('allows to read output on exception', function* () {
+      let { stdout, stderr } = yield exec("node './test/fixtures/hello-world-failed.js'").expect();
```

Comment:
This should, in fact, never return because it raises an error. It's the same as `yield Promise.reject()` 

`expect()` means that you expect it to succeed and will fail if it doesn't. If you change this to `exec().join()` it will wait for the process to finish, whatever the result and return the process result.

I think the answer here is to include a portion of `stdout` and `stderr` into the exception so that you can quickly diagnose it.